CS.Security/DTO/UserRoles.cs

```diff
@@ -0,0 +1,10 @@
+namespace CS.Security.DTO;
+
+public static class UserRoles
```

Comment:
I found that you build your logic based on a string literals in a multiple places. This is a dangerous way, as any single typo could break the code and make the finding of a bug extremely difficult. So usually we take string literals into constants like this and use 'strongly typed' values in the code